server/plugin/webhook.go

```diff
@@ -935,15 +948,6 @@ func (p *Plugin) postPullRequestReviewCommentEvent(event *github.PullRequestRevi
 			continue
 		}
 
-		post := p.makeBotPost(newReviewMessage, "custom_git_pull_review_comment")
-
-		repoName := strings.ToLower(repo.GetFullName())
-		commentID := event.GetComment().GetID()
-
-		post.AddProp(postPropGithubRepo, repoName)
-		post.AddProp(postPropGithubObjectID, commentID)
-		post.AddProp(postPropGithubObjectType, githubObjectTypePRReviewComment)
-
 		post.ChannelId = sub.ChannelID
 		if err = p.client.Post.CreatePost(post); err != nil {
```

Comment:
Referencing https://github.com/mattermost/mattermost-plugin-github/pull/768/files#r1649049447 the `p.client.Post.CreatePost(post)` here mutates the post instead of returning the created post, which has had side effects in this plugin. I think we should instead use `p.API.CreatePost(post)` to avoid the risk of introducing the issue on unrelated code changes cc @hanzei